tests/RuleSet/SampleRulesBad.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\RuleSet;
+
+/**
+ * Sample external RuleSet.
+ *
+ * This class is not extending the required class `\PhpCsFixer\RuleSet\RuleSetDescriptionInterface`,
+ * so it will not be a valid class to be registered as a RuleSet.
+ */
+class SampleRulesBad
```

Comment:
should it be a class we store in drive, or simply an anonymously created class we setup directly in tests preparation?

Reply:
It could be anonymous class probably. And there was [comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6083#discussion_r1232378279) from @kubawerlos about moving this file to fixtures dir, so it should be either way.

Reply:
This is now resolved with moving the files to fixtures. Personally I prefer to test with existing files like these.